### PR TITLE
Fix flex issues in png dashboard export (#7889)

### PR DESCRIPTION
**Pull request by Fiery-Fenix**
#7889

### CODE REVIEW

File: tasks/options/postcss.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = function(config) {
       },
 
       processors: [
-        require('autoprefixer')({browsers: 'last 3 versions'}), // add vendor prefixes
+        require('autoprefixer')({browsers: ['last 3 versions', 'safari >= 4']}), // add vendor prefixes
```
Comment (torkelo) [inline tasks/options/postcss.js]: Safari 4.1.3 was released in	2005 ?

Comment (Fiery-Fenix) [inline tasks/options/postcss.js]: This workaround is recommended in related [phantomjs issue](https://github.com/ariya/phantomjs/issues/14365) and it works like expected.

Comment (torkelo) [inline tasks/options/postcss.js]: yes, but not sure I want add add this to prefixer, unsure what that will do as it's such an old browser .

**Comment by torkelo** [comment]
Closing this as taking png snapshots of whole dashboards is not fully supported yet. And if we do it as a real feature I think dashboard links will likely be hidden anyway

**Pull request closed without merging**